transfer.md

```diff
@@ -1,76 +1,48 @@
-| `docker cp`     | [`podman mount`](https://podman.readthedocs.io/en/latest/markdown/podman-mount.1.html) ****   |
-| `docker create` | [`podman create`](https://podman.readthedocs.io/en/latest/markdown/podman-create.1.html)  |
-| `docker diff`   | [`podman diff`](https://podman.readthedocs.io/en/latest/markdown/podman-diff.1.html)      |
-| `docker export` | [`podman export`](https://podman.readthedocs.io/en/latest/markdown/podman-export.1.html)  |
-| `docker history`| [`podman history`](https://podman.readthedocs.io/en/latest/markdown/podman-history.1.html)|
-| `docker images` | [`podman images`](https://podman.readthedocs.io/en/latest/markdown/podman-images.1.html)  |
-| `docker kill`   | [`podman kill`](https://podman.readthedocs.io/en/latest/markdown/podman-kill.1.html)      |
-| `docker load`   | [`podman load`](https://podman.readthedocs.io/en/latest/markdown/podman-load.1.html)      |
-| `docker login`  | [`podman login`](https://podman.readthedocs.io/en/latest/markdown/podman-login.1.html)    |
-| `docker logout` | [`podman logout`](https://podman.readthedocs.io/en/latest/markdown/podman-logout.1.html)  |
-| `docker pause`  | [`podman pause`](https://podman.readthedocs.io/en/latest/markdown/podman-pause.1.html)    |
-| `docker ps`     | [`podman ps`](https://podman.readthedocs.io/en/latest/markdown/podman-ps.1.html)          |
-| `docker pull`   | [`podman pull`](https://podman.readthedocs.io/en/latest/markdown/podman-pull.1.html)      |
-| `docker push`   | [`podman push`](https://podman.readthedocs.io/en/latest/markdown/podman-push.1.html)      |
-| `docker rename` | [`podman rename`](./docs/podman-rename.1.md)  |
-| `docker rm`     | [`podman rm`](https://podman.readthedocs.io/en/latest/markdown/podman-rm.1.html)          |
-| `docker rmi`    | [`podman rmi`](https://podman.readthedocs.io/en/latest/markdown/podman-rmi.1.html)        |
-| `docker run`    | [`podman run`](https://podman.readthedocs.io/en/latest/markdown/podman-run.1.html)        |
-| `docker save`   | [`podman save`](https://podman.readthedocs.io/en/latest/markdown/podman-save.1.html)      |
-| `docker stop`   | [`podman stop`](https://podman.readthedocs.io/en/latest/markdown/podman-stop.1.html)      |
-| `docker tag`    | [`podman tag`](https://podman.readthedocs.io/en/latest/markdown/podman-tag.1.html)        |
-| `docker unpause`| [`podman unpause`](https://podman.readthedocs.io/en/latest/markdown/podman-unpause.1.html)|
-| `docker version`| [`podman version`](https://podman.readthedocs.io/en/latest/markdown/podman-version.1.html)|
-| `docker wait`   | [`podman wait`](https://podman.readthedocs.io/en/latest/markdown/podman-wait.1.html)   |
-
-*** Use `podman exec` to enter a container and `podman logs` to view the output of pid 1 of a container.
-**** Use mount to take advantage of the entire linux tool chain rather then just cp.  Read [`here`](https://podman.readthedocs.io/en/latest/markdown/podman-cp.1.html) for more information.
```

Comment:
Are the removed sections, system tools, equivalents, and development transfer really no longer useful?

Reply:
System tools and Equivalents have a few things which are still useful. I will update the doc accordingly.